test/functional/pipeline.js

```diff
@@ -12,7 +12,7 @@ describe('pipeline', function () {
         [null, 3],
         [null, '3']
       ]);
-      redis.disconnect()
+      redis.disconnect();
```

Comment:
Don't add `;` here. Code style changes should belong to a separate pull request.

Reply:
resolved